CODEOWNERS

```
@@ -4,3 +4,4 @@
 /internal/provider/cqlauth/        @rjeczalik
 /internal/provider/serverless/     @rjeczalik
 /internal/provider/vpcpeering/     @rjeczalik
+/internal/provider/connection/     @rjeczalik @dkropachev
```

```suggestion
/internal/provider/connection/     @charconstpointer @dkropachev
```

Can you please send this change as a separate PR?

---

Done